backend/spec/controllers/spree/admin/stock_transfers_controller_spec.rb

```diff
@@ -24,6 +24,26 @@ module Spree
           transfer.closed_at = DateTime.now
         end }
 
+      describe "stock location filtering" do
+        let(:user) { create(:admin_user) }
+        let(:ability) { Spree::Ability.new(user) }
+        let!(:sf_store) { StockLocation.create(name: "SF Store")}
+
+        before do
+          ability.cannot :manage, Spree::StockLocation
```

Comment:
i wish https://github.com/solidusio/solidus/pull/449 was ready, b/c it could clean up some of this spec setup